src/components/Person/Person.jsx

```diff
@@ -1 +1,27 @@
-// export const Person = ({ person }) => ();
+import './Person.scss';
+
+export const Person = ({ person }) => (
+  <div className="App">
+    <section className="Person">
+      <h2 className="Person__name">
+        {`My name is ${person.name}`}
+      </h2>
+
+      { person.age !== undefined && (
+      <p className="Person__age">
+        {`I am ${person.age}`}
+      </p>
+      )}
+
+      <p className="Person__partner">
+        {!person.isMarried
+          ? `I am not married`
+          : `${person.partnerName} is my ${
+            person.sex === 'f'
+              ? `husband`
+              : `wife`
+          }`}
```

Comment:
using nested ternary operators is bad practice
Let's move this logic to a separate method